### DISCUSSION TITLE
The Licence of Linol

### DISCUSSION THREAD

**Discussion by SoilihBenSoilih** [c-cube/linol#4] [General]
Hello, 

Hope you're doing well !
I'm using Linol in my project and I want to know the licence of Linol to include it in my project.
It writen `See file "LICENSE"` in the template but I didn't find it. I want to use **GPLv3+** but 
I need to be sure that the licence of Linol permit me to do that.

Thank you a lot !!!

**Comment by c-cube** [comment]
I just wrote in the readme that's it's MIT licensed. It's compatible with GPLv3 :)
